js/views/site.es6.js

```diff
@@ -16,6 +18,16 @@ function Site (ops) {
       [this.$whitelisttoggle, 'click', this._whitelistClick]
     ]);
 
+    // FIXME should be moved to the right place --- model?
```

Comment:
I agree this is probably not the right place for this. Do you want to figure that out now or deal with it later on though?